src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const male = people
+    .filter(human => ((Math.ceil(human.died / 100) === century)
+      && (human.sex === 'm')) || (human.sex === 'm' && !century));
```

Comment:
something strange, use ternary operators to make it code more readable, fix in all cases